crypto/Cargo.toml

```diff
@@ -11,10 +11,9 @@ default = ["std"]
 # Enable static linkage of the rust standard library.
 # Please refer to https://docs.rust-embedded.org/book/intro/no-std.html
 std = ["ursa"]
+ffi = ["iroha_ffi"]
```

Comment:
I have to introduce this feature flag temporarily because `iroha_ffi` requires `std`. The only reason it requires `std` is because of `catch_unwind`